backend/src/test/java/com/zeros/backend/service/RouteServiceImplTest.java

```diff
@@ -0,0 +1,35 @@
+package com.zeros.backend.service;
+
+import java.math.BigDecimal;
+import java.time.Duration;
+
+import org.assertj.core.api.Assertions;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.test.context.ActiveProfiles;
+import org.springframework.test.context.junit4.SpringRunner;
+
+import com.google.maps.model.TravelMode;
+import com.zeros.backend.BackendApplication;
+
+@SpringBootTest(classes = BackendApplication.class)
+@ActiveProfiles("ut")
+@RunWith(SpringRunner.class)
+public class RouteServiceImplTest {
```

Comment:
Why you have crate a ut profile ?

Reply:
to separate the test conf from the envs conf